### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by xeviff** [Magisk-Modules-Repo/com.piyushgarg.rclone#93]
Hi.
I'm trying to get it work. Rooted Nvidia Shield, installed Magisk, configured remotes on `/sdcard/.rclone/rclone.conf`, put `BINDSD=1` on `.*.param` in order to get non root visibility from sdcard by other applications.... But I can't get it work.
This is after I uninstalled a previous installation because of it wasn't creating the proper folders (`/sdcard/cloud` and etc).
Now when I install the package again I get "**_Installation failed_**" error, without futher details:
![image](https://user-images.githubusercontent.com/73612508/118311774-9f23b400-b4f0-11eb-9c3a-5f7f1b56f782.png)
(could I get some logs of that anywhere else?)

The rclone command is still available on the terminal, but it doesn't work with the proper path (which should be: `/sdcard/.rclone/rclone.conf`, but for some reason it tries to find it on `/data/data/com.termux/files/home/.suroot/.config/rclone/rclone.conf`)
![image](https://user-images.githubusercontent.com/73612508/118312641-f1190980-b4f1-11eb-8210-c953dee749ee.png)

I have no idea why it's not getting the configurations, and specially why I can't do a clean reinstallation. Do you have some reasons of all this happenings?

I wiped all data and configurations of my 3 years old Nvidia Shield just for use rclone on it. I hope it gets succesfull at the end :-)
I think this project is a wonderful idea, if it works in my machine at the end, for sure I'll do a little contribution for you.

**Comment by JedDeLagged** [comment]
Had the same problem, just sorted it out.

Short Answer, delete/remove your sdcard/.rclone/rclone.conf

Long Answer, there's an issue with mounting your configurations, either by erroneious configuration or fault with the mounting process.

Get rid of just that file (or make sure the only file is .global.param), and it should install. Worked for me.

**Comment by xeviff** [comment]
Thanks for the response. I uninstalled (because spide of the error message, the app was installed anyway) + reinstalled after deleting the file and leaving sonly .global.param file.
Now the installation is complaining about the missing file:
![image](https://user-images.githubusercontent.com/73612508/121777483-a04d0d00-cb92-11eb-8b90-86988b8008d8.png)

After this, I executed `rclone config` and the message says the same rare path (`/data/data/com.termux` instead of `/sdcard/.rclone/rclone.conf`)

This time, I did some diferent (because I don't have the .rclone.config file): Configure manually a remote by `rclone config `command. 
![image](https://user-images.githubusercontent.com/73612508/121778103-b0b2b700-cb95-11eb-87ba-1a2bb32a6220.png)

Then I restarted, and now I don't find the `/mnt/cloud` and either `/sdcard/cloud`
![image](https://user-images.githubusercontent.com/73612508/121778172-09824f80-cb96-11eb-84eb-0fe16dc3261d.png)

In the previous scenario (with sdcard/.rclone/rclone.conf file) at least the app was mounting the remotes, Only what I needed is to be visible from `/sdcard/cloud` (with no root ownership, visible for the Android apps, like Kodi for example).

What a mess... I'm sorry hehe
